### ISSUE TITLE
x0f.org

### ISSUE THREAD

**Issue by TheBlueMatt** [hachyderm/community#165]
Hi!

Apologies if you saw this before, I only just now saw the README here which says "dont DM, and dont email", both of which were easier to find :).

I’m reaching out to ask why hachyderm has blocked x0f.org. x0f is basically developers-only (it’s a personal+friends) instance with a number of members who work actively on Rust cryptocurrency projects and post about Rust things (the Rust community appears to have found a home with hacyderm).

I don't know all the members, but know a good chunk of them to be incredibly reasonable, often somewhat liberal, individuals. The instance is also well-moderated, insofar as they actually have issues. Thus, I'm left wondering why it was banned. The instance admin indicated he only learned about it indirectly (and was surprised/saddened to learn it had been banned anywhere) and assumed it was because most of the members are developers interested in/working on cryptocurrency projects.

I see hachyderm had previously banned bitcoinhackers.org, but now appears to have un-banned that, and I can say with a lot of confidence bitcoinhackers.org hosts some...controversial figures, whereas x0f very much does not.

**Comment by quintessence** [comment]
👋 Hello, mod here :) 

So for your first question: we curate our blocklist mostly based on 1 ) other publicized blocklists, focusing on servers whose ethos would violate our server rules and/or 2 ) a domain that has been reported to us for doing so. We also have some servers that are limited due to being focused on bots / other inorganic forms of high volume traffic.

For this particular one, we have looked into the server and it looks like it is overall in good faith; however, I did take a look at the post you linked. Instances that are blocked and believe they have been blocked in error are welcome to reach out to the moderator team to ask us to look into the situation and ask questions, as you have done here. That said, we don't respond to venting to the ether as a "Call to Action" if that makes sense.
